modules/migration-about-configuring-proxies.adoc

```diff
@@ -131,3 +131,34 @@ spec:
       cidrSelector: <cidr_of_source_or_target_cluster>
     type: Deny
 ----
+
+[id="configuring-supplemental-groups-for-rsync-pods{context}"]
+=== Configuring supplemental groups for Rsync pods
+When your PVCs use shared storage, you can configure the access to storage by adding supplemental groups to Rsync pod definitions in order for the pods to allow access:
+
```

Comment:
@RichardHoch  I would think it should be '**the** storage' or '**that** storage'. Isn't it the same shared storage that the PVCs use? I would even go as far as to say "...PVCs use **a** shared storage", treating 'storage' as a countable noun because here it obviously means an object - a particular server or drive or volume. Feel free to disagree.